pom.xml

```diff
@@ -34,7 +34,7 @@
   </licenses>
 
   <properties>
-    <raml-module-builder.version>27.1.1</raml-module-builder.version>
+    <raml-module-builder.version>28.1.1</raml-module-builder.version>
     <vertx.version>3.8.1</vertx.version>
```

Comment:
please remove this explicit dependency on vertx
vertx should be a transitive dependency that comes wtih RMB

Reply:
mod-kb-ebsco-java is using some modules that are not present in RMB, for example vertx-unit, vertx-service-proxy, also it is using vertx modules directly, so I think it is ok to have direct dependency on vertx